Formula/stlink.rb

```diff
@@ -15,15 +15,13 @@ class Stlink < Formula
   end
 
   depends_on "libusb"
+  depends_on "gtk+3" => :optional
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  depends_on "cmake" => :build
```

Comment:
It's a fine rule, but it's not easily discoverable and I guess it's easy to forget to remove the empty line before `head` again when the `revision` line gets removed due to a version bump.